lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_all_uk_students.govspeak.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title do %>
-  Do any of the following apply (you might get extra funding)?
+  Do any of the following apply (You might get extra funding)?
```

Comment:
Same, with the `eg` on line 7 instead.

Reply:
Change any eg you find to "for example" please.

Reply:
Thanks. Done!